NGitLab.Mock/Repository.cs

```diff
@@ -50,11 +50,27 @@ private LibGit2Sharp.Repository GetGitRepository()
                         var directory = TemporaryDirectory.Create();
                         if (Project.ForkedFrom == null)
                         {
+                            LibGit2Sharp.Repository.Init(directory.FullPath);
+
                             // libgit2sharp cannot init with a branch other than master
-                            using var process = Process.Start("git", $"init --initial-branch \"{Project.DefaultBranch}\" \"{directory.FullPath}\"");
+                            // Use symbolic-ref to keep the code compatible with older version of git
+                            using var process = new Process
+                            {
+                                StartInfo = new ProcessStartInfo
+                                {
+                                    FileName = "git",
+                                    Arguments = $"-C \"{directory.FullPath}\" symbolic-ref HEAD refs/heads/{Project.DefaultBranch}",
```

Comment:
You can use `WorkingDirectory` instead of `-C` to make it easier to understand.

Also you should quote the last parameter